package.json

```diff
@@ -11,7 +11,7 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
-    "axios": ">=0.18.1",
+    "axios": "^0.21.1",
```

Comment:
?

Reply:
If I remember well, @freearhey got the same changes.